src/providers/WorkflowCore.Providers.Elasticsearch/WorkflowCore.Providers.Elasticsearch.csproj

```diff
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
```

Comment:
Do we need to update the Elastic package to support the EF upgrade?

Reply:
Same answer as the others - we can try it but it seems worse to have different versions of the library in use